Add `__riscv_` prefix to the intrinsics in example programs

Abdulwadoodd opened the pull request:
Following PR #189 , the example programs have been updated by adding the `__riscv_` prefix to all the intrinsic. 
I have tested these programs by compiling them using risc-v clang toolchain and executing them on the qemu emulator using the following command

`qemu-riscv64 -L $RISCV_TOOLCHAIN_ROOT/sysroot/ -cpu rv64,v=true,zba=true,vlen=128 test.o`


All the programs give the `pass` status upon execution on qemu, except the `rvv_branch.c`. I'm unable to figure out why it is failing. Kindly help me with that.

eopXD commented:
Wow! Thank you for watching out for closely watching the changes of this repository. I really appreciate this.

In https://github.com/riscv-non-isa/rvv-intrinsic-doc/pull/186, we have simplified the intrinsic and fused the two separate sets (non-policy / policy) into a single set. Now the default policy for a masked instructions is tail agnostic and mask agnostic. To achieve the original code, you need to specify the `_mu` and provide the `maskedoff` argument.

```
--- a/examples/rvv_branch.c
+++ b/examples/rvv_branch.c
@@ -21,7 +21,7 @@ void branch(double *a, double *b, double *c, int n, double constant) {

     vbool64_t mask = __riscv_vmfne_vv_f64m1_b64(vec_b, vec_zero, vl);

-    vfloat64m1_t vec_c = __riscv_vfdiv_vv_f64m1_m(mask, vec_a, vec_b, vl);
+    vfloat64m1_t vec_c = __riscv_vfdiv_vv_f64m1_mu(mask, /*maskedoff*/ vec_constant, vec_a, vec_b, vl);
     __riscv_vse64_v_f64m1(c, vec_c, vl);
   }
 }
```

Please add `Signed-off-by: YOUR NAME <YOUR EMAIL>` in the commit, thank you!

Abdulwadoodd commented:
> 
> In #186, we have simplified the intrinsic and fused the two separate sets (non-policy / policy) into a single set. Now the default policy for a masked instructions is tail agnostic and mask agnostic. To achieve the original code, you need to specify the `_mu` and provide the `maskedoff` argument.
> 
> ```
> --- a/examples/rvv_branch.c
> +++ b/examples/rvv_branch.c
> @@ -21,7 +21,7 @@ void branch(double *a, double *b, double *c, int n, double constant) {
> 
>      vbool64_t mask = __riscv_vmfne_vv_f64m1_b64(vec_b, vec_zero, vl);
> 
> -    vfloat64m1_t vec_c = __riscv_vfdiv_vv_f64m1_m(mask, vec_a, vec_b, vl);
> +    vfloat64m1_t vec_c = __riscv_vfdiv_vv_f64m1_mu(mask, /*maskedoff*/ vec_constant, vec_a, vec_b, vl);
>      __riscv_vse64_v_f64m1(c, vec_c, vl);
>    }
>  }
> ```

Thanks for the help. 
Now `rvv_branch.c` program gives the `pass` status along with all the other example programs. 

Updated the changes.

eopXD commented:
Verified on my end too. Thank you for the quick update.
